The only real coder here/ruby oop/dog_lion_mammal/lion.rb

```diff
@@ -0,0 +1,26 @@
+require_relative "mammal"
+class Lion < Mammal
+	def initialize(name)
+		super(@mammal_name)
+		@health += 20
+	end
+	def fly
+		@health -= 10
+		self
+	end
+	def attack_town
+		@health -= 50
+		self
+	end
+	def eat_humans
+		@health += 20
+		self
+	end
+	def lion_display
```

Comment:
so the assignment actually wants you to name this display_health just like in the mammal class, but as you probably figured out that "breaks" how you want it to work. BUT, you can utilize super to still call the mammal display_health method.